packages/bridge/src/imports/module.ts

```diff
@@ -91,7 +92,7 @@ export default defineNuxtModule<Partial<ImportsOptions>>({
     await regenerateImports()
 
     // Generate types
-    addDeclarationTemplates(ctx)
+    addDeclarationTemplates(ctx, options)
```

Comment:
```suggestion
    addDeclarationTemplates(ctx)
```

Reply:
The `options` are used here!

https://github.com/nuxt/bridge/blob/dfd7beee88e306ceef165a24500d71fe801a9e9b/packages/bridge/src/imports/module.ts#L144-L151

Fixed to be the same as Nuxt 3. Is it unnecessary?
https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/imports/module.ts#L161-L168

Reply:
you're quite right!